hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/XceiverServerGrpc.java

```diff
@@ -172,6 +173,11 @@ public void start() throws IOException {
   public void stop() {
     if (isStarted) {
       server.shutdown();
+      try {
+        server.awaitTermination(1, TimeUnit.DAYS);
```

Comment:
It's better to read the value from configuration file.

Reply:
Same question as above.